style.css

```diff
@@ -0,0 +1,647 @@
+  justify-content: end;
+  align-items: center;
+  height: 84px;
+  background-image: url("/images/launch_bg.gif");
+  background-repeat: no-repeat;
+  margin-bottom: 1rem;
+}
+
+.main__content__banner a {
+  display: inline-block;
+  background-image: url("/images/view_bg.gif");
+  width: 58px;
+  height: 44px;
+  color: white;
+  font-family: var(--fnt-btn-primary);
+  font-weight: bold;
+  text-decoration: none;
+  font-size: 0.8rem;
+  text-align: center;
+  margin-right: 1rem;
+  padding: 0.3rem;
+}
+
+/* ////////// CONTACT FORM //////////// */
+
+.contact__container {
+  max-height: 380px;
+  width: 237px;
+}
+
+.contact__container::before,
+.contact__container::after {
+  display: inline-block;
+  content: "";
+  width: 237px;
+  height: 3px;
+}
+.contact__container::before {
+  background-image: url("/images/contact_top_curve.gif");
+}
+
+.contact__container::after {
+  background-image: url("/images/contact_bottom_curve.gif");
+}
+
+.contact__form__header {
+  background-image: url("/images/contact_bg.gif");
+  background-repeat: no-repeat;
+  height: 39px;
+  display: flex;
+  flex-shrink: 0;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 0.5rem;
+  padding-left: 2.2rem;
+  width: 191px;
+}
+
+.contact__form__header h3,
+.contact__form__container form > label {
+  color: var(--clr-primary-100);
+}
+
+.contact__form__container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--clr-primary-600);
+  width: 100%;
+  padding: 1rem 0 1rem;
+}
+
+.contact__form__container form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding: 0.3rem 1.5rem 0.6rem;
+  width: 100%;
+  gap: 0.2rem;
+}
+
+.contact__form__container textarea {
+  resize: none;
+}
+
+.contact__form__button__container {
+  display: flex;
+  justify-content: end;
+  gap: 0.5rem;
+  margin-top: 1rem;
+}
+
+.contact__form__button__container button {
+  width: 64px;
+  line-height: 19px;
+}
+
+.contact__form__button__container button:first-of-type {
+  background-image: url("/images/reset_bg.gif");
+}
+.contact__form__button__container button:last-of-type {
+  background-image: url("/images/submit_bg.gif");
+}
+
+/* ////////////// FOOTER SECTION /////////////// */
+
+footer {
+  width: 100vw;
+  height: 153px;
+  background-color: #000000;
+  display: flex;
+  justify-content: center;
+}
+
+.footer__logo {
+  background-image: url("/images/footer_logo.gif");
+  width: 328px;
+  height: 127px;
+}
+
+.footer__items {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: start;
+}
+
+.footer__nav {
+  margin: 1.5rem 0 1rem 1rem;
+}
+
+.footer__nav ul {
+  list-style: none;
+}
+
+.footer__nav li {
+  float: left;
+  margin-right: 0.75rem;
+}
+
+.footer__nav li::after {
+  content: "|";
+  color: white;
+  padding-left: 0.75rem;
+}
+
+.footer__nav li:last-of-type::after {
+  content: none;
+}
+
+.footer__nav a {
+  color: white;
```

Comment:
When using colors, best to always use hex codes or rgb or rgba values. SO in this case, `#fff`

Reply:
Noted. I think I adjusted all the named colors to hex values now.